samples/.NET/MSTest/Tests.cs

```diff
@@ -0,0 +1,52 @@
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Playwright;
+using Microsoft.Playwright.MSTest;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace PlaywrightTests;
+
+[TestClass]
+public class Tests : PageTest
+
+{
+    [TestMethod]
+
+    public async Task HomepageHasPlaywrightInTitleAndGetStartedLinkLinkingtoTheIntroPage()
+    {
+        
+       var context = await Browser.NewContextAsync();
+
+        // // Start tracing.
+        // await context.Tracing.StartAsync(new()
+        // {
+        //     Screenshots = true,
+        //     Snapshots = true,
+        //     Sources = true
+        // });
+        
+        var page = await context.NewPageAsync();
+        await page.GotoAsync("https://playwright.dev");
```

Comment:
since we are using PageTest, we should just use Page.Goto instead of creating context within test and then create page.